docs/tutorials/tabular_prediction/tabular-indepth.md

```diff
@@ -46,89 +58,76 @@ gbm_options = { # specifies non-default hyperparameter values for lightGBM gradi
 
 ```{.python .input}
-test_data = val_data.copy()
-y_test = test_data[label_column]
-test_data = test_data.drop(labels=[label_column],axis=1)  # delete label column
-
-y_pred = predictor.predict(test_data)
+y_pred = predictor.predict(test_data_nolabel)
 print("Predictions:  ", list(y_pred)[:5])
 perf = predictor.evaluate_predictions(y_true=y_test, y_pred=y_pred, auxiliary_metrics=False)
 ```
 
-Use the following to view a summary of what happened during fit. This command will shows details of the hyperparameter-tuning process for each type of model:
+Use the following to view a summary of what happened during fit. Now this command will show details of the hyperparameter-tuning process for each type of model:
 
 ```{.python .input}
 results = predictor.fit_summary()
 ```
 
-In the above example, the predictive performance may be poor because we specified very little training to ensure quick runtimes.  You can call `fit()` multiple times while modifying the above settings to better understand how these choices affect performance outcomes. For example: you can comment out the `train_data.head` command to train using a larger dataset, increase the `time_limits`, and increase the `num_epochs` and `num_boost_round` hyperparameters.  To see more detailed output during the execution of `fit()`, you can also pass in the argument: `verbosity = 3`.
-
-
-## Specifying performance metrics
-
-Performance in certain applications may be measured by different metrics than the ones AutoGluon optimizes for by default. If you know the metric that counts most in your application, you can specify it as done below to utilize the balanced accuracy metric instead of standard accuracy (the default):
-
-```{.python .input}
-metric = 'balanced_accuracy'
-predictor = task.fit(train_data=train_data, label=label_column, eval_metric=metric,
-                     output_directory=output_directory, time_limits=60)
-
-performance = predictor.evaluate(val_data)
-```
-
-Some other non-default metrics you might use include things like: `f1` (for binary classification), `roc_auc` (for binary classification), `log_loss` (for classification), `mean_absolute_error` (for regression), `median_absolute_error` (for regression).  You can also define your own custom metric function, see examples in the folder: `autogluon/utils/tabular/metrics/`
+In the above example, the predictive performance may be poor because we specified very little training to ensure quick runtimes.  You can call `fit()` multiple times while modifying the above settings to better understand how these choices affect performance outcomes. For example: you can comment out the `train_data.head` command to train using a larger dataset, increase the `num_epochs` and `num_boost_round` hyperparameters, and increase the `time_limits` (which you should do for all code in these tutorials).  To see more detailed output during the execution of `fit()`, you can also pass in the argument: `verbosity = 3`.
```

Comment:
I would suggest setting a named variable in the tutorial for the `sample_size` that you can refer to here along with the other variables you mention